.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Build release docs
+
+on:
+    # Allows you to run this workflow manually from the Actions tab
+    # It should be run right after tagging the release commit on main
+    workflow_dispatch:
+
+jobs:
+    release-docs:
+        # if: github.repository == 'nilearn/nilearn' && contains('["ymzayek","Remi-Gau"]', github.actor)
+        if: github.repository == 'ymzayek/nilearn' && github.actor == 'ymzayek'
```

Comment:
This will have to evolve ...

Reply:
It's just for testing on my fork and being extra careful

Reply:
I don't like to have stuff that we will rot quite quickly and repeatedly...

Reply:
I'm not sure I understand. But before merging this it will only have 
`if: github.repository == 'nilearn/nilearn' `

Reply:
In fact I'll already make the changing since I'm testing on my fork